src/minitests/mod.rs

```diff
@@ -46,77 +48,122 @@ fn test_from_file(path: &Path) {
 
 #[test]
 fn test_arraytest() {
-    test_from_file(Path::new("builtin/arraytest.mexe"));
+    test_from_file(
+        Path::new("builtin/arraytest.mexe"),
+        Value::Buffer(Buffer::new_empty()),
+    );
 }
 
 #[test]
 fn test_kvstest() {
-    test_from_file(Path::new("builtin/kvstest.mexe"));
+    test_from_file(
+        Path::new("builtin/kvstest.mexe"),
+        Value::Buffer(Buffer::new_empty()),
+    );
 }
 
 #[test]
 fn test_storage_map() {
-    test_from_file(Path::new("stdlib/storageMapTest.mexe"));
+    test_from_file(
+        Path::new("stdlib/storageMapTest.mexe"),
+        Value::Buffer(Buffer::new_empty()),
+    );
 }
 
 #[test]
 fn test_queuetest() {
-    test_from_file(Path::new("stdlib/queuetest.mexe"));
+    test_from_file(
+        Path::new("stdlib/queuetest.mexe"),
+        Value::Buffer(Buffer::new_empty()),
+    );
 }
 
 #[test]
 fn test_globaltest() {
-    test_from_file(Path::new("builtin/globaltest.mexe"));
+    test_from_file(
+        Path::new("builtin/globaltest.mexe"),
+        Value::Buffer(Buffer::new_empty()),
+    );
 }
 
 #[test]
 fn test_pqtest() {
-    test_from_file(Path::new("stdlib/priorityqtest.mexe"));
+    test_from_file(
+        Path::new("stdlib/priorityqtest.mexe"),
+        Value::Buffer(Buffer::new_empty()),
+    );
 }
 
 #[test]
 fn test_bytearray() {
-    test_from_file(Path::new("stdlib/bytearraytest.mexe"));
+    test_from_file(
+        Path::new("stdlib/bytearraytest.mexe"),
+        Value::Int(Uint256::from_usize(0)),
+    );
 }
 
 #[test]
 fn test_map() {
-    test_from_file(Path::new("builtin/maptest.mexe"));
+    test_from_file(
+        Path::new("builtin/maptest.mexe"),
+        Value::Buffer(Buffer::new_empty()),
```

Comment:
Added in my latest commit -- the plan really is for every test to return some kind of string, but since that won't always be the case until we add more kinds string functionality, having that wrapper makes sense.